src/mon/OSDMonitor.cc

```diff
@@ -11394,7 +11394,7 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
       err = 0;
       goto reply;
     }
-    ceph_assert(osdmap.require_osd_release >= ceph_release_t::luminous);
+    ceph_assert(osdmap.require_osd_release >= ceph_release_t::octopus);
```

Comment:
should we change this assert to nautilus in pacific?

Reply:
yeah. we could. but this does not change the behavior of a pacific cluster. it improves the readability though.